src/compiler/xslt/compile/compile-scenario.xsl

```diff
@@ -147,9 +128,9 @@
 
             <!-- Handle local preceding variable declarations and apply/call/context in document
                order, instead of apply/call/context first and variable declarations second. -->
```

Comment:
Can you delete "apply/" in two spots within this comment?

Reply:
Done in 0e209a5a534a63ce74856f5105a2108ac4549203.